### PR TITLE
Update from Travis to Github Actions

### PR DESCRIPTION
**Pull request by juliusgeo**

### CODE REVIEW

**Comment by juliusgeo** [comment]
https://github.com/juliusgeo/pymongoexplain/actions/runs/1551489377 Here is the github actions run. All tests appear to be passing.

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: pymongoexplain
+
+on: [push, pull_request]
+
+jobs:
+  build:
```
Comment (ShaneHarvey) [inline .github/workflows/test.yml:6]: Should this be called "Test" instead of "Build"?

**Review by ShaneHarvey — APPROVED** [review]
LGTM with one comment

File: .github/workflows/test.yml (hunk 1 above)
Comment (juliusgeo) [reply .github/workflows/test.yml]: Done.

**Pull request merged**